minimesos/src/test/java/com/containersol/minimesos/DefaultMinimumClusterTest.java

```diff
@@ -38,8 +38,7 @@
 
     @After
     public void after() {
-        DockerContainersUtil util = new DockerContainersUtil(dockerClient);
-        util.getContainers(false).filterByName(HelloWorldContainer.CONTAINER_NAME_PATTERN).kill().remove();
+        cluster.destroy();
```

Comment:
I don't think we have any control of the containers that are started by schedulers (ie we can't give them the name prefixed with minimesos-<CLUSTERID>) and that's the only way for us to identify if the containers belongs to a cluster.

So, for this case it's suffice to call cluster.destroy(), and not a single test starts schedulers that start containers, so I don't think it's a problem